x-pack/plugins/security_solution/public/network/components/network_top_n_flow_table/index.tsx

```diff
@@ -112,11 +112,16 @@ const NetworkTopNFlowTableComponent: React.FC<NetworkTopNFlowTableProps> = ({
     [sort, dispatch, type, tableType]
   );
 
-  const field =
-    sort.field === NetworkTopTablesFields.bytes_out ||
-    sort.field === NetworkTopTablesFields.bytes_in
-      ? `node.network.${sort.field}`
-      : `node.${flowTargeted}.${sort.field}`;
+  const field = useMemo(
```

Comment:
nit -> instead of having two useMemo
```
const sorting = useMemo(() => ({
  field: sort.field === NetworkTopTablesFields.bytes_out ||
      sort.field === NetworkTopTablesFields.bytes_in
        ? `node.network.${sort.field}`
        : `node.${flowTargeted}.${sort.field}`, 
      direction: sort.direction }
), [flowTargeted, sort]);
```